portafly/src/tests/components/pages/accounts/AccountsTable.test.tsx

```diff
@@ -8,21 +8,25 @@ import {
   useDataListBulkActions
 } from 'components'
 import { AccountsTable, generateColumns, generateRows } from 'components/pages/accounts'
-import { developerAccounts } from 'tests/examples'
-import { fireEvent, within } from '@testing-library/react'
+import { within } from '@testing-library/react'
 import { IDeveloperAccount } from 'types'
+import { factories } from 'tests/factories'
 
 jest.mock('components/shared/data-list/DataListContext')
 const useTableMock = useDataListTable as jest.Mock
 const usePaginationMock = useDataListPagination as jest.Mock
 const useFiltersMock = useDataListFilters as jest.Mock
 const useBulkActionsMock = useDataListBulkActions as jest.Mock
+const developerAccounts: IDeveloperAccount[] = [['Dandelion', 'Rosemary and Thyme'], ['Geralt', 'Wolf School']].map((account) => (
+  factories.DeveloperAccount.build({
+    admin_name: account[0],
+    org_name: account[1]
+  })
+))
```

Comment:
If theme can be avoided 😂 I would suggest using `buildList` instead for less verbosity:
```suggestion
const developerAccounts = factories.DeveloperAccount.buildList(2)

Reply:
I know, but I really like themed tests!!! it makes me happy